composer.json

```diff
@@ -13,9 +13,9 @@
     "require": {
         "php": ">=7.0",
         "dragonbe/vies": "^1.0",
-        "symfony/http-kernel": "^2.8",
-        "symfony/symfony": "^2.8",
-        "symfony/validator": "^2.8"
+        "symfony/http-kernel": "^2.8|^3.0|^4.0",
+        "symfony/dependency-injection": "^2.8|^3.0|^4.0",
```

Comment:
single pipe is de oude OR operator, de voorkeur wordt nu gegeven aan double pipe (`||`)  zie : https://getcomposer.org/doc/articles/versions.md#version-range

Reply:
hoe zit het met de volgorde waarin deze version geresolved worden? kan er geen documentatie over vinden

Reply:
Dat is afhankelijk van het project waarbij jij deze library gaat opnemen. Composer resolved alle requirements en pakt de best matching versie. Als library ansich, wellicht, minder interessant. Desondanks beweren we nu dat deze dat code compatible is met Symfony `2.x`, `3.x` en `4.x`.